.github/workflows/ci.yml

```diff
@@ -53,7 +53,7 @@ jobs:
       fail-fast: false
       matrix:
         php-versions: ['7.2', '7.3', '7.4']
```

Comment:
```suggestion
        php-versions: ['7.2', '7.3', '7.4', '8.0']
```

Reply:
Whoops! Just noticed we didn't have PHP 8 in our test matrix. We'll also need to do the same in other `php-verions` matrix in this file